content/en/docs/instrumentation/java/manual.md

```diff
@@ -704,18 +704,47 @@ instrument.
 
 ### Auto Configuration
 
-To configure the OpenTelemetry SDK based on the standard set of environment
-variables and system properties, you can use the
-`opentelemetry-sdk-extension-autoconfigure` module.
+Instead of manually creating the `OpenTelemetry` instance by using the SDK builders
+directly from your code, it is also possible to use the SDK auto-configuration extension 
+through the `opentelemetry-sdk-extension-autoconfigure` module.
+
+This module is made available by adding the following dependency to your application.
+
+```xml
+<dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-exporter-jaeger</artifactId>
```

Comment:
is this the right artifactId? From the text I would assume that it is `opentelemetry-sdk-extension-autoconfigure`?

Reply:
Absolutely not!! :-D
Going to fix it!